<?xml version="1.0"?>
<launch>
     


     <arg name="dbg_args"   default="gdb -ex run --args"/>
     <arg name="debug_args"   default=""/>

     <node pkg="stair_detection" type="correlation_sample_with_svm_ros" name="correlation_sample_with_svm_ros_node" launch-prefix="$(arg debug_args)" output="screen">

<!-- anchor_factor_index 1: gradient anchor  2 : gradient diff anchor  3: depth y error sum anchor*/ -->
          <param name="anchor_factor_index"      value="3"/>
          <param name="result_save_dir"        value="/home/kangneoung/stair_detection/src/stair_detection/sample_correlation"/>
          <param name="delta_gradient"        value="0.05"/>
          <param name="delta_gradient_diff"      value="0.01"/>
          <param name="delta_depth_y_error_sum"      value="0.2"/>
          <param name="gradient_range_min"    value="0.3"/> 
          <param name="gradient_range_max"        value="1.8"/>
          <param name="gradient_diff_range_min"        value="0.15"/>
          <param name="gradient_diff_range_max"          value="0.5"/>
          <param name="depth_y_error_sum_range_min"        value="0.2"/>
          <param name="depth_y_error_sum_range_max"        value="4"/>
       
     </node>

</launch>
